doc/development.rst

```diff
@@ -625,9 +625,32 @@ Code
 
 Documentation
 ~~~~~~~~~~~~~
+
 - Use `semantic linefeeds
   <http://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_ in .rst files.
 
+Building the documentation
+++++++++++++++++++++++++++
+
+When contributing to documentation it's practical to be able to build it also locally
```

Comment:
You missed a full stop here at the end.

Reply:
Assuming "full stop" is the "." at the end of the sentence. 
:+1: Fixing